pyproject.toml

```diff
@@ -19,17 +19,8 @@ public = true
 [tool.irt.publish.rpm]
 rpm_repo_name_prefix = "inmanta/oss"
 
-[tool.irt.publish.docker_image.dev]
-image_registry = "code.inmanta.com:4567"
-image_repository = "solutions/containers"
-image_name = "orchestrator"
-image_tags = [
-    "dev",
-]
-registry_username_env = "GITLAB_REGISTRY_USERNAME"
-registry_password_env = "GITLAB_REGISTRY_PASSWORD"
```

Comment:
If we move this to the native image, we need to update the documentation as well.

Reply:
Can I defer this to https://github.com/inmanta/inmanta-core/issues/8304 as well?